app/views/development_metrics/repositories.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="row">
   <div class="col-md-12 text-center mt-2">
-    <b><%= @repository.product[:name] %></b>
+    <b><%= @repository.product[:name] if @repository.product.present? %></b>
```

Comment:
how is this change related to code coverage? usually I'd recommend creating separate PRs for unrelated changes, but we can leave it this time

Reply:
you are right! 
It is not related, I found this bug when navigating the web department report by repository and fix it, as it was one line I commit it with this issue. My bad.

Reply:
As I will leave it this time, I fixed the PR description to have quick reference for this change.